net/transmission/files/transmission.init

```diff
@@ -154,11 +154,10 @@ transmission() {
 	[ -d "$web_home" ] && procd_set_param env TRANSMISSION_WEB_HOME="$web_home"
 
 	procd_add_jail transmission log
-	procd_add_jail_mount "$config_file"
+	procd_add_jail_mount_rw "$config_dir"
 	procd_add_jail_mount_rw "$config_dir/resume"
 	procd_add_jail_mount_rw "$config_dir/torrents"
 	procd_add_jail_mount_rw "$config_dir/blocklists"
```

Comment:
These lines aren't necessary anymore since you already mounted their parent in r/w mode.